### PR TITLE
Enhance find feature

### LINKED ISSUE #35: As a ward nurse, I can find patients via their NRIC
so that I can quickly find their information from the list of patients.

### PR DESCRIPTION
**Pull request by swtan346**
find by either name or IC. Only single IC for each find command.
Command only support finding by name or IC as this functionality is expected to search for specific patient.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
 import seedu.address.logic.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.Person;
 
 /**
  * Finds and lists all persons in address book whose name contains any of the argument keywords.
- * Keyword matching is case insensitive.
+ * Keyword matching is case-insensitive.
  */
```
Comment (iamtr) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:17]: Can you update this such that it reflects the ways we can use "find" as well?

**Review by iamtr — COMMENTED** [review]
Code LGTM to me. Just the docs needed to be updated as well.

Perhaps can you update the description of the PR with more detail? Such as why not find with ward, and/or examples

File: src/main/java/seedu/address/logic/commands/FindCommand.java (hunk 1 above)
Comment (swtan346) [reply src/main/java/seedu/address/logic/commands/FindCommand.java:17]: This is stated in message usage.

**Review by iamtr — APPROVED** [review]
LGTM

**Pull request merged**